crates/kitsune_p2p/kitsune_p2p/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+- **BREAKING** - AgentInfo uses a quantized arc instead of an arc half-length to represent DHT coverage. This is a breaking protocol change, nodes on different versions will not be able to gossip with each other.
```

Comment:
Should we bump that protocol version that makes the dna hashes different?

Reply:
I would love to, but it still doesn't exist yet :/ it's in #3463, which is almost about to be merged